File: python/Monitor.py
class Monitor:
    def __init__(self, nama="", ukuran_layar="", aspek_rasio="", resolusi="", tipe_panel="", refresh_rate=0):
        self.nama = nama
        self.ukuran_layar = ukuran_layar
        self.aspek_rasio = aspek_rasio
        self.resolusi = resolusi
        self.tipe_panel = tipe_panel
        self.refresh_rate = refresh_rate
    
    def setNama(self, nama):
        self.nama = nama
    
    def setUkuranLayar(self, ukuran_layar):
        self.ukuran_layar = ukuran_layar
    
    def setAspekRasio(self, aspek_rasio):
        self.aspek_rasio = aspek_rasio
    
    def setResolusi(self, resolusi):
        self.resolusi = resolusi
    
    def setTipePanel(self, tipe_panel):
        self.tipe_panel = tipe_panel
    
    def setRefreshRate(self, refresh_rate):
        self.refresh_rate = refresh_rate
    
    def getNama(self):
        return self.nama
    
    def getUkuranLayar(self):
        return self.ukuran_layar
    
    def getAspekRasio(self):
        return self.aspek_rasio
    
    def getResolusi(self):
        return self.resolusi
    
    def getTipePanel(self):
        return self.tipe_panel
    
    def getRefreshRate(self):
        return self.refresh_rate

File: cpp/Monitor.cpp
#pragma once
#include <iostream>
#include <string>
using namespace std;

class Monitor
{
private:
    string nama;
    string ukuran_layar;
    string aspek_rasio;
    string resolusi;
    string tipe_panel;
    int refresh_rate;

public:
    Monitor() {}

    Monitor(string nama, string ukuran_layar, string aspek_rasio, string resolusi, string tipe_panel, int refresh_rate)
    {
        this->nama = nama;
        this->ukuran_layar = ukuran_layar;
        this->aspek_rasio = aspek_rasio;
        this->resolusi = resolusi;
        this->tipe_panel = tipe_panel;
        this->refresh_rate = refresh_rate;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    void setUkuranLayar(string ukuran_layar)
    {
        this->ukuran_layar = ukuran_layar;
    }

    void setAspekRasio(string aspek_rasio)
    {
        this->aspek_rasio = aspek_rasio;
    }

    void setResolusi(string resolusi)
    {
        this->resolusi = resolusi;
    }

    void setTipePanel(string tipe_panel)
    {
        this->tipe_panel = tipe_panel;
    }

    void setRefreshRate(int refresh_rate)
    {
        this->refresh_rate = refresh_rate;
    }

    string getNama()
    {
        return nama;
    }

    string getUkuranLayar()
    {
        return ukuran_layar;
    }

    string getAspekRasio()
    {
        return aspek_rasio;
    }

    string getResolusi()
    {
        return resolusi;
    }

    string getTipePanel()
    {
        return tipe_panel;
    }

    int getRefreshRate()
    {
        return refresh_rate;
    }

    ~Monitor() {}
};